lib/src/components/NxCloseButton/NxCloseButton.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-colors';
+@import '../../scss-shared/nx-container-helpers';
+
+.nx-btn--close {
+  border-color: transparent;
+
+  .nx-icon {
+    height: $nx-spacing-md;
+    width: $nx-spacing-md;
+  }
```

Comment:
We should not use spacing variables for height/width. They should be kept separate.

Reply:
> We should not use spacing variables for height/width. They should be kept separate.

What variables should I be using here?

Reply:
> What variables should I be using here?

Actually, I guess what I should really do is set the height to `1em` and then just use the font size, which is already conveniently the 16px we want.

Reply:
> > What variables should I be using here?
> 
> Actually, I guess what I should really do is set the height to `1em` and then just use the font size, which is already conveniently the 16px we want.

Or I was going to say that you should use `$nx-font-size`

Reply:
> Actually, I guess what I should really do is set the height to 1em and then just use the font size, which is already conveniently the 16px we want.

25270990